test/e2e/devconsole.operatorsource.cr.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: devconsole-operator
+  namespace: openshift-operators
+spec:
+  type: appregistry
+  endpoint: https://quay.io/application/ldimaggi-test-org/mychart
```

Comment:
Nice! Do paste a screenshot in this PR how it looks in the operatorhub page on the cluster.

Reply:
I spent a lot of time with Helm this morning and I *think* I have the correct sequence - but while the subscription is showing up inthe UI, the catalog is not. The problem that I am seeing is:  https://github.com/operator-framework/operator-lifecycle-manager/issues/822 - currently blocked on this.